target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -170,6 +171,10 @@ ipq40xx_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
+	telco,x1pro-lte)
+		base_mac=$(mtd_get_mac_binary "0:ART" 0x5006)
+		lan_mac=$(macaddr_add "$base_mac" 2)
+		wan_mac=$(macaddr_add "$base_mac" 3)
```

Comment:
Missing ";;"

Reply:
Both should be set in DTS, though.

Reply:
And nothing in 0x0 or 0x6?

Reply:
0x0 and 0x6 have the radio identifiers, this is how it was done by the factory

Reply:
>Both should be set in DTS, though.

Done, Please check the latest version.

Reply:
ping @adschm

Reply:
https://github.com/openwrt/openwrt/blob/6f96a4d043a9367c6c0d166299d808df764e88e6/target/linux/ath79/dts/ar9344_tplink_tl-wdr3500-v1.dts#L59

Reply:
For this, should I use `&gmac` instead of &eth?